docs/standard/serialization/system-text-json/populate-properties.md

```diff
@@ -20,33 +27,75 @@ class A
 }
 ```
 
-When you create an instance of this class, the `Numbers1` (and `Numbers2`) property's value is a list with three elements (1, 2, and 3). If you deserialize JSON to this type, the default behavior is that property values are replaced:
+When you create an instance of this class, the `Numbers1` (and `Numbers2`) property's value is a list with three elements (1, 2, and 3). If you deserialize JSON to this type, the *default* behavior is that property values are *replaced*:
 
-- For `Numbers1`, since it's read-only (no setter), it will still have the values 1, 2, and 3 in its list.
-- For `Numbers2`, which is read-write, the contents of the list will be replaced with the values from the JSON.
+- For `Numbers1`, since it's read-only (no setter), it still has the values 1, 2, and 3 in its list.
+- For `Numbers2`, which is read-write, the contents of the list are effectively replaced with the values from the JSON.
 
 For example, if you execute the following deserialization code, `Numbers1` contains the values 1, 2, and 3 and `Numbers2` contains the values 4, 5, and 6.
 
 ```csharp
 A? a = JsonSerializer.Deserialize<A>("""{"Numbers1": [4,5,6], "Numbers2": [4,5,6]}""");
 ```
 
-Starting in .NET 8, you can change the deserialization behavior to modify (*populate*) initialized properties and fields instead of replacing them. And since the value that the property references isn't *replaced* (that is, doesn't require a setter), if the type of the property is mutable, deserialization can also populate *read-only* properties.
+## Populate behavior
 
-The <xref:System.Text.Json.Serialization.JsonObjectCreationHandling> enum fields provide the object creation handling choices:
+Starting in .NET 8, you can change the deserialization behavior to modify (*populate*) initialized properties and fields instead of replacing them:
 
-- <xref:System.Text.Json.Serialization.JsonObjectCreationHandling.Replace?displayProperty=nameWithType> (matches the default behavior)
-- <xref:System.Text.Json.Serialization.JsonObjectCreationHandling.Populate?displayProperty=nameWithType>
+- For a collection type property, any existing values are kept, and new values from the JSON are added to the collection.
+- For a non-collection reference type property, its mutable properties are updated to the JSON value but the reference itself doesn't change.
+- For a struct type property, the effective behavior is that for its mutable properties, the existing values are kept, and new values from the JSON are added. However, unlike a reference property, the object itself isn't reused since it's a value type. Instead, a copy of the struct is modified and then reassigned to the property. For example, after executing the following code, `a.S1.Numbers3` contains the values 1, 2, 3, 4, 5, and 6.
```

Comment:
It might make sense to explicitly mention that populating struct properties requires having a setter (which isn't the case for reference types).